src/go/MONIT/ggus_alerting.go

```diff
@@ -273,9 +277,15 @@ func deleteAlerts() {
 		temp.Annotations.Scope = each.Annotations.Scope
 		temp.Annotations.URL = each.Annotations.URL
 
-		temp.StartsAt = each.StartsAt
+		emp.StartsAt = each.StartsAt
```

Comment:
are you sure that `emp` is an appropriate name here?

Reply:
No, it was a mistake.

Reply:
next time try to compile your code before submitting the PR since the compiler will fetch this kind of typos.

Reply:
Yes. I will follow this. 
Thanks